packages/flame/lib/src/text/common/glyph_info.dart

```diff
@@ -0,0 +1,14 @@
+/// Helper class that stores dimensions of a single glyph for a spritesheet-
+/// based font.
+class GlyphInfo {
```

Comment:
`GlyphDimensions` maybe? It's hard to differentiate `GlyphInfo` and `GlyphData` with their current names.

Reply:
Good point, we should probably merge these two somehow.
However, since this PR didn't create these classes -- it merely moved them out of `sprite_font_renderer.dart` file -- we should probably defer this refactoring to a separate PR.

Reply:
Will you remember it? Or should we add an issue to track it with?

Reply:
I think we'll remember. We would have to do some refactoring around those classes anyways, in order to support non-monospace sprite fonts.